xc7/fasm2bels/net_map.py

```diff
@@ -0,0 +1,72 @@
+""" Utilities for match VPR route names with xc7 site pin sources. """
+from collections import namedtuple
+from lib.parse_route import find_net_sources
+import re
+"""
```

Comment:
This file has two docstrings?

Reply:
The first docstring is for module, the second docstring is for the Net namedtuple.

Reply:
For namedtuples how would you feel about this pattern:
```python
class Net(namedtuple('Net', 'name wire_pkey tile site_pin')):
    """
    Args:
        name (str): VPR net name
        wire_pkey (int): Wire table primary key.  This is unique in the part.
        tile (str): Name of tile this wire belongs too.  This is redundant
            information wire_pkey uniquely indentifies the tile.
        site_pin (str): Name of site pin this wire belongs. This is redundant
            information wire_pkey uniquely indentifies the site pin.
    """
```

Reply:
It appears to work the same, so I don't mind it.